lib/webpacker/configuration.rb

```diff
@@ -80,10 +84,13 @@ def load
       end
       config[env].deep_symbolize_keys
     rescue Errno::ENOENT => e
-      raise "Webpacker configuration file not found #{config_path}. " \
-            "Please run rails webpacker:install " \
-            "Error: #{e.message}"
-
+      if self.class.installing
```

Comment:
@guillaumebriday @dhh I think this is a better way to plumb the setting of "installing."

Let me know if there is a better way.